lib/core/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ generate_export_header(${TARGET_NAME})
 target_link_libraries(
   ${TARGET_NAME} PUBLIC scipp-common scipp-units Boost::Boost Eigen3::Eigen
 )
-if(TBB_FOUND AND NOT DISABLE_MULTI_THREADING)
+if(TBB_FOUND)
```

Comment:
See previous comment. Use `if(THREADING)` instead?

Reply:
I kept this, since it is explicitly linking to TBB::tbb, so I think `if(TBB_FOUND)` is clearer and makes sense.